consensus/consensus_v2.go

```diff
@@ -368,12 +368,12 @@ func (consensus *Consensus) Start(
 					if k != timeoutViewChange {
 						consensus.getLogger().Warn().Msg("[ConsensusMainLoop] Ops Consensus Timeout!!!")
 						viewID := consensus.GetCurBlockViewID()
```

Comment:
Can directly use consensus.GetCurBlockViewID(). can you help remove viewID local var?

Reply:
sure.

Reply:
can remove the viewID param for consensus.startViewChange() actually as the value used in inside consensus.

Reply:
done.